net/adguardhome/Makefile

```diff
@@ -56,7 +56,7 @@ endef
 define Build/Compile
 	( \
 		pushd $(PKG_BUILD_DIR) ; \
-		make js-deps js-build ; \
+		NODE_OPTIONS=--openssl-legacy-provider make js-deps js-build ; \
```

Comment:
`export NODE_OPTIONS=--openssl-legacy-provider ; \`
`make js-deps js-build ; \`

I see they recommend this writing style, and it works.

Reply:
There's no difference in this case.

Reply:
Just to make it look more comfortable...

Reply:
It's just a personal preference. tbh I don't like making local variables global.